src/libraries/System.Net.Security/src/System/Net/Security/SslConnectionInfo.cs

```diff
@@ -5,6 +5,12 @@ namespace System.Net.Security
 {
     internal partial struct SslConnectionInfo
     {
+#pragma warning disable CA1823
+        private static readonly byte[] s_http1 = SslApplicationProtocol.Http11.Protocol.ToArray();
+        private static readonly byte[] s_http2 = SslApplicationProtocol.Http2.Protocol.ToArray();
+        private static readonly byte[] s_http3 = SslApplicationProtocol.Http3.Protocol.ToArray();
```

Comment:
This feels like an opportunity to use the utf8 string literal feature.


cc @stephentoub